src/main/java/com/revature/autosurvey/surveys/controllers/SurveyController.java

```diff
@@ -53,8 +53,12 @@ public void setSurveyService(SurveyService surveyService) {
 
 	@DeleteMapping("{id}")
 	public Mono<ResponseEntity<Object>> deleteSurvey(@PathVariable("id") UUID uuid) {
-		return surveyService.deleteSurvey(uuid)
-				.map(survey -> ResponseEntity.noContent().build())
+		return surveyService.deleteSurvey(uuid).map(survey -> ResponseEntity.noContent().build())
 				.onErrorResume(error -> Mono.just(ResponseEntity.notFound().build()));
 	}
+
+	@GetMapping(path = "all")
```

Comment:
Wait, why are we adding a GET".../surveys/all" path? Why not just have this functionality under GET".../surveys/" ?

Reply:
Because I forgot that getByID uses the ID at the end. Will update.